src/components/spatial/Spatial.js

```diff
@@ -4,16 +4,89 @@ import {
   ScatterplotLayer, PolygonLayer, COORDINATE_SYSTEM, BitmapLayer, BaseTileLayer,
 } from 'deck.gl';
 import { load } from '@loaders.gl/core';
-import { SelectablePolygonLayer } from '../../layers';
+import { Texture2D } from '@luma.gl/webgl';
+import GL from '@luma.gl/constants';
+import { slice, openArray } from 'zarr';
+import { SelectablePolygonLayer, XRLayer } from '../../layers';
 import { tileToBoundingBox, getTileIndices } from './tiling-utils';
 import { cellLayerDefaultProps, PALETTE, DEFAULT_COLOR } from '../utils';
 import AbstractSelectableComponent from '../AbstractSelectableComponent';
 import LayersMenu from './LayersMenu';
 
+const zarrArrays = {};
```

Comment:
This is used for caching - once we formalize the API for the data layer more strongly, all of this fetching/caching will be its own thing.  For now, this will do for the purposes of demoing.